scripts/milestone.py

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env python3
+
+import datetime
+import logging
+import os
+import sys
+from itertools import groupby
+
+import click
+import requests
```

Comment:
I'm a little surprised `isort` is not compaining here about missing group headers, will have to investigate. Also will need to add `click` as dependency in conda environment yaml files

Reply:
Updated and added 👍